sun-asterisk-research/table_reconstruction
# test: add test case for table detection module

## Description
Add some test cases for table detection module
remove unused functions of table detection module


## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?
Firstly, check module table detection after refactor
- Create and  activate new python environment, after that install all dependencies via command `pip3 install -r requirements.txt`
-  Download 2 files [table_file.pdf](https://drive.google.com/file/d/1ZsFKeUNd5qNngTzl_spLVympa2Xyn40f/view?usp=sharing) and [table_detection.ipynb](https://drive.google.com/file/d/1wjkGgUIn6UAFdGbnKURs2uXvNED8ORVN/view?usp=sharing), put them in the root directory of the repo.
- run Jupyter notebook by using the command `jupyter notebook`. then open **table_detection.ipynb** and run all cells.
Check results of the module at the last cell of the notebook.

After that, check test
`coverage run --source table_reconstruction -m unittest discover tests`
`coverage html`

- check coverage in `htmlcov` folder. Open file `index.html` to see the coverage report.

## Checklist

- [x] I have successfully run the modified code in my local environment with no errors
- [x] I ran the proper linters and fixed the issues pointed out to make sense
- [x] I updated the unit test based on my edit
